sympy/printing/latex.py

```diff
@@ -120,7 +120,7 @@
 
 _between_two_numbers_p = (
     re.compile(r'[0-9][} ]*$'),  # search
-    re.compile(r'[0-9]'),  # match
+    re.compile(r'(\d|\\frac{\d+}{\d+})'),  # match
```

Comment:
Is there any test case already covering the number multiplied with fraction?

Reply:
Yes, and they were corrected by the commit [#2a797a8b93f15f9dbf02609f3ec4e1d1057a92a1](https://github.com/sympy/sympy/pull/21101/commits/2a797a8b93f15f9dbf02609f3ec4e1d1057a92a1).

Reply:
Okay, it looks like it corrects some confusing patches introduced by that PR.